src/libraries/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Compile.cs

```diff
@@ -149,5 +150,929 @@ public void TFS_470021()
                 Assert.Equal(0, errorCount);
             }
         }
+
+        /// <summary>
+        /// Test for issue #30218, resource Sch_MinLengthGtBaseMinLength
+        /// </summary>
```

Comment:
NIT: we don't really add documentation for test methods, normally method names illustrate what it does, and your method name expressing what it testing very well, please remove the comment.

Reply:
Removed.

Reply:
Somehow still can see it 😄, maybe forgotten?

Reply:
Sorry.  I scrolled through the file, deleted two of them, and mistakenly thought that was all....

It's gone now.  And Ctrl+F shows there aren't any more I missed.

Reply:
No problem at all, and thanks!